config/connection_settings.go

```diff
@@ -133,14 +139,27 @@ func CreateAzureCertificateConnectionSettings(
 	return connSettings, nil
 }
 
-// CreateAzureProvisioningConnectionSettings creates the configuration data for establishing connection to Azure device that requires device provisioning.
+// CreateAzureProvisioningConnectionSettings creates the configuration data for establishing connection to Azure device that requires device provisioning, using no IDScopeProvider.
```

Comment:
typo `connection to Azure device` should be `connection to Azure IoT Hub`

Reply:
This typo is from the initial contribution of the source code, I suppose there are other typos too and I would suggest to open another ticket for fixing such issues.

Reply:
@stoyan-zoubev - any news here? Did you have time to check for some more typos? I cannot see new related issues in the project. In any case, I would suggest to fix what's found in the current PR.